### DISCUSSION TITLE
get error API rate limit exceeded for installation ID ***

### DISCUSSION THREAD

**Discussion by Rosyya** [community/community#43814] [Actions]
### Select Topic Area

Product Feedback

### Body

we are using self hosted runners, and in workflow there is a public action `actions/github-script@v3` with `secrets.GITHUB_TOKEN` for authenticated requests, from official document: 'At the start of each workflow run, GitHub automatically creates a unique GITHUB_TOKEN secret to use in your workflow. ', but still get api rate limit exceeded error.
my workflow like:
```
- uses: actions/github-script@v3
   with:
     github-token: ${{ secrets.GITHUB_TOKEN }}
     script: |
       ......
       ......
```

**Comment by jameshounshell** [comment]
Did you ever find a solution for this? We're also using self hosted runners but using a temporary token created from a github apps creds.
